byway/src/main/webapp/destinations.js

```diff
@@ -132,15 +131,37 @@ function addMarker(searchBox,map){
 /** 
 * fetches start location and destinations from DestinationsServlet and adds to DOM
 */
-function updateLocations(locationData){
-    document.getElementById('start-location').innerText = "Start Location :" + locationData.start;
+function updateLocations(locationData, map){
     const container = document.getElementById('destinations-container');
-    container.innerText = "Destinations:";
+    container.innerText="";
```

Comment:
Add space around =
Same as in lines 147, 149

Reply:
done